src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -35,10 +34,16 @@ public static class OpenTelemetryServicesExtensions
     /// cref="IServiceCollection"/>.
     /// </summary>
     /// <remarks>
-    /// Note: This is safe to be called multiple times and by library authors.
+    /// Notes:
+    /// <list type="bullet">
+    /// <item>This is safe to be called multiple times and by library authors.
```

Comment:
not something added in this PR, but this is *not* to be called by library authors....so must be a typo here? @CodeBlanch do you think this was intended?

Reply:
There's really 2 questions...

Q: Is it safe for library authors to call this?

A: Yes! Multiple calls will stack. Everything is building-up/modifying a single provider for any given IServiceCollection. This is what the comments are saying.

Q: Should library authors call this?

A: It depends! If you want to force OpenTelemetry to be started, perfectly fine to call this. If you just want to configure something in the event OTel happens to be used by the host, don't call this. That is what `IServiceCollection.ConfigureOpenTelemetry[Tracer|Meter|Logger]Provider` extensions are for. The current comments aren't really addressing that bit, I'll follow-up on that.